contrib/images/simd-env/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && \
+  apt-get -y upgrade && \
+  apt-get -y install curl jq file
+
+ARG UID=1000
+ARG GID=1000
+
+USER ${UID}:${GID}
+VOLUME [ /simd ]
+WORKDIR /simd
+EXPOSE 26656 26657
+ENTRYPOINT ["/usr/bin/wrapper.sh"]
+CMD ["start"]
+STOPSIGNAL SIGTERM
+
+COPY wrapper.sh /usr/bin/wrapper.sh
```

Comment:
@alessio were you able to get it to work when the contrib/images was prefixed here? It works like this locally but not in ci

Reply:
ignore, resolved in a different manner